docs/UserGuide.adoc

```diff
@@ -273,37 +273,123 @@ Examples:
 
 Create and add a new adminstrator account. +
-Format: `addAdmin user\<USERNAME> pass\<PASSWORD> n\<NAME> ptp\<PATH_TO_PIC> salary\<SALARY> date\<EMPLOYMENTDATE>`
+Format: `addAdmin OPTION/INPUT...` +
 
-Examples:
+Input fields:
+
+* Username : `user/<USERNAME>`
+* Password : `pass/<PASSWORD>`
+* Name : `n/<NAME>`
+* Save path : `sp/<SAVE_PATH>`
+* Salary : `salary/<SALARY>`
+* Employment date : `date/<EMPLOYMENTDATE>`
+
+Example:
 
-* `addAdmin user\master pass\myPass n\John Doe ptp\johnDoe.img salary\4000 date\01/01/2018` +
+* `addAdmin user/master pass/myPass#1 n/John Doe sp/userconfig.xml salary/4000 date/01/01/2018` +
 Creates a new administrator account with the following stated details.
 
-=== Remove a existing administrator account: `removeAdmin` `[coming in v2.0]`
+=== Edit the currently logged in administrator account: `editAdmin`
 
-Remove a currently existing administrator account. +
-Format: `removeAdmin <USERNAME>`
+Edit the details of the currently logged in administrator. +
+Format: `editAdmin OPTION/INPUT...` +
+
+Input fields:
+
+* Name : `n/[NAME]`
+* Salary : `salary/[SALARY]`
+* Employment date : `employ/[EMPLOYMENT_DATE]`
 
 Examples:
 
-* `removeAdmin user\adminUsername` +
-Removes a administrator account with username "adminUsername" if it exists.
+* `editAdmin n/john`
+Change the name of the logged in administrator to "john".
+* `editAdmin salary/3000 employ/01/01/2018`
+Change both the salary and employement date of the logged in administrator to "3000" and "01/01/2018" respectively.
+
+=== Remove a existing user account: `removeUser`
 
-=== Add a new module to database: `addModuleDB` `[coming in v1.2]`
+Remove a currently existing user account. +
+Format: `removeAdmin <USERNAME>`
+
+Example:
+
+* `removeAdmin Username` +
+Removes a administrator account with username "Username" if it exists.
+
+=== Show the list of usernames: `showUser`
+
+Display the list of usernames on the main display. +
+Format: `showUser`
+
+=== Add a new module to database: `addModuleDB`
 
 Creates a new module and add it to the module database. +
-Format: `addModuleDB code\<MODULE_CODE> department\<DEPARTMENT> title\<TITLE> credit\<CREDIT> available\<SEM1><SEM2><SPECIAL_TERM1><SPECIAL_TERM2> description\<DESCRIPTION>`
+Format: `addModuleDB OPTION/INPUT...` +
 
-Examples:
+Input fields:
 
-* `addModuleDB code\CS2109 department\Computing title\The New Module credit\4 available\yynn description\This course is newly offered by the school of computing. Visit the module website for more details` +
-Create and add a new module with module code "CS2109" and the stated details to the module database.
+* Module code : `code/<MODULE_CODE>`
+* Department : `department/<DEPARTMENT>`
+* Title : `title/<TITLE>`
+* Description : `description/<DESCRIPTION>`
+* Credit : `credit/<CREDIT>`
+* Available semesters : `available/<SEM1><SEM2><SPECIAL_TERM1><SPECIAL_TERM2>`
++
+[NOTE]
+For each entry enter '1' if available, '0' if not
```

Comment:
It would be better to rephrase `For each entry enter '1' if available, '0' if not` to `For each entry enter '1' if xxx is available or '0' otherwise`

Reply:
Changes applied.